hermes-console/static/partials/topic.html

```diff
@@ -120,10 +120,9 @@ <h3 class="panel-title">Properties</h3>
 
     <div class="row" ng-show="topic.contentType == 'AVRO'">
         <div class="col-md-12">
-            <div class="panel panel-default">
-                <div style="position: relative">
-                    <span class="btn pull-right glyphicon {{showMessageSchema ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down' }}" ng-click="showMessageSchema = !showMessageSchema"></span>
-                </div>
+            <div class="panel panel-default" style="position: relative">
+                <span class="btn pull-right glyphicon {{showMessageSchema ? 'glyphicon-chevron-up' : 'glyphicon-chevron-down' }}" ng-click="showMessageSchema = !showMessageSchema"></span>
+                <button class="btn btn-default pull-right" style="margin-top: 2px" ng-click="copyToClipboard()">Copy to clipboard</button>
```

Comment:
![image](https://user-images.githubusercontent.com/17440354/96366380-b0b60080-1147-11eb-9cf4-856d86757883.png)
Or we can add smaller padding, icnrease margin top to have:
![image](https://user-images.githubusercontent.com/17440354/96366413-c5929400-1147-11eb-8122-e03e08c14fe1.png)
Let me now which one is better for you

Reply:
It's better with padding and margin-top.

Reply:
![image](https://user-images.githubusercontent.com/17440354/96827116-231f2d00-1435-11eb-816d-331ae943b9cf.png)
I've adjusted padding and margin-top a little bit. 

Please re-approve. Thank you!